vars/jobs.groovy

```diff
@@ -38,3 +38,13 @@ def build_job(name, parameters, is_stable)
   }
   return job
 }
+
+def build_and_notify(name, parameters, ignore_unstable=false){
+  def current_job = build job: name, parameters: parameters, propagate: false, wait: true
+  if (current_job.result != 'SUCCESS') {
+    if (env.NODE_NAME.contains('flow') || (ignore_unstable && current_job.result != 'UNSTABLE')) {
+      notification.notificator(env.BUILD_URL, get_job_url(current_job))
+    }
+    throw new Exception(other.parse_url(get_job_url(current_job), true) + " has TERMINATED with status " + current_job.result)
```

Comment:
I would also add whole exception message from exception we got.

error: Error signal
Signals an error. Useful if you want to conditionally abort some part of your program. You can also just throw new Exception(), but this step will avoid printing a stack trace.

As you wanted to print just this exception with Error step we will miss stack trace which is useful for debugging